src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.tsx

```diff
@@ -28,35 +28,73 @@ const EndorsementChainLayoutUnstyled: FunctionComponent<EndorsementChainLayout>
+          if (!beneficiaryChangeEvent.holderChangeEvents || !beneficiaryChangeEvent.beneficiary)
+            return new Error("Invalid Event: Transfer Event does not have new beneficiary or new holder address");
+          beneficiaryChangeEvent.holderChangeEvents.forEach((holderChangeEvent, holderIndex) => {
+            tableRows.push(
+              <tr className="table-row" key={index++}>
+                <td className="table-cell date">
+                  {format(new Date(holderChangeEvent.timestamp), "do MMM yyyy, hh:mm aa")}
+                </td>
+                <td className="table-cell endorsement-ui-dash">
+                  {eventIndex === 0 && holderIndex === 0 && <div className="mask" />}
+                  <AddressCell
+                    address={beneficiaryChangeEvent.beneficiary}
+                    titleEscrowAddress={beneficiaryChangeEvent.documentOwner}
+                    newAddress={!(previousBeneficiary === beneficiaryChangeEvent.beneficiary)}
+                  />
+                </td>
+                <td className="table-cell endorsement-ui-dash">
+                  {eventIndex === 0 && holderIndex === 0 && <div className="mask" />}
+                  <AddressCell
+                    address={holderChangeEvent.holder}
+                    titleEscrowAddress={beneficiaryChangeEvent.documentOwner}
+                    newAddress={!(previousHolder === holderChangeEvent.holder)}
+                  />
+                </td>
+              </tr>
+            );
+            previousBeneficiary = beneficiaryChangeEvent.beneficiary;
+            previousHolder = holderChangeEvent.holder;
+          });
+          return;
+        case "Surrender":
+          tableRows.push(
+            <tr className="table-row" key={index++}>
+              <td className="table-cell date border-top-none">
+                {format(new Date(tradetrustErc721Event?.eventTimestamp ?? 0), "do MMM yyyy, hh:mm aa")}
+              </td>
+              <td colSpan={2} className="endorsement-ui-dash border-top-none">
+                <div className="name-row">
+                  <div className="dot" data-testid="dot" />
+                  <div className="name">Document surrendered to Issuer</div>
+                </div>
+              </td>
+            </tr>
+          );
+          return;
+        case "Burnt":
+          tableRows.push(
+            <tr className="table-row" key={index++}>
+              <td className="table-cell date border-top-none">
+                {format(new Date(tradetrustErc721Event?.eventTimestamp ?? 0), "do MMM yyyy, hh:mm aa")}
+              </td>
+              <td colSpan={2} className="endorsement-ui-dash border-top-none">
+                <div className="name-row">
+                  <div className="dot" data-testid="dot" />
+                  <div className="name">Surrender of document accepted</div>
+                </div>
+              </td>
+            </tr>
+          );
+          return;
```

Comment:
break;

Also, doesnt linter complain when there is no default case?

Reply:
nope but ill add it in just in case